common/protos/index.ts

```diff
@@ -50,3 +50,35 @@ function fromBase64(value: string): Uint8Array {
   util.base64.decode(value, buf, 0);
   return buf;
 }
+
+export class ProtoUtils {
```

Comment:
sorry, i think i wasn't clear. this file already contains all of these utilities, i don't think we need `ProtoUtils` at all.